mods/common/languages/en.ftl

```diff
@@ -1,3 +1,11 @@
+## GameSpeed
```

Comment:
```suggestion
## MapOptions, MissionBrowserLogic
```

?

Reply:
Yes, correct. That was my first attempt where `ModData` was still `null`.